templates/donor-donation-refund-full.twig

```diff
@@ -0,0 +1,17 @@
+{% extends renderHtml ? '_base.html.twig' : '_base.text.twig' %}
+
+{% block content %}
+  <p>Dear {{ donorFirstName }},</p>
+
+  <p>The following payment made by you has now been refunded:</p>
+
+  <ul>
+    <li>Donation reference: {{donationReferenceNumber}}</li>
```

Comment:
Ideally it would be good to have spacing in these interpolations consistently so it's easier to search for usages etc.